rust/kernel/error.rs

```diff
@@ -102,11 +117,14 @@ impl fmt::Debug for Error {
             fn rust_helper_errname(err: c_types::c_int) -> *const c_types::c_char;
         }
         // SAFETY: FFI call.
-        let name = unsafe { rust_helper_errname(-self.0) };
+        let name = unsafe { rust_helper_errname(-self.to_kernel_errno()) };
```

Comment:
Is the `to_kernel_errno()` call required here? Why not use `-self.0.get()` ?

Reply:
> Is the `to_kernel_errno()` call required here? 

no

> Why not use `-self.0.get()` ?

To me, using `to_kernel_errno()` is clearer, and if we change the inner representation of Error again in the future, we don't need to touch this code. But that is just my personal taste. :)